src/components/RadonOperator.vue

```diff
@@ -128,8 +129,9 @@ export default {
       variables: state => state.rad.currentTemplate.variables,
     }),
     radonError() {
+      console.log(this.error)
```

Comment:
Is this `console.log` intentional?

In any case, we have `console.error` now :fireworks: 
```suggestion
      console.error(this.error)
```

Reply:
It should be removed anyway, but thanks!